spaceship/lib/spaceship/test_flight/client.rb

```diff
@@ -70,19 +70,15 @@ def put_build(provider_id, app_id, build_id, build)
     end
 
     def post_for_review(provider_id, app_id, build_id, build)
-      begin
-        response = request(:post) do |req|
-          req.url "providers/#{provider_id}/apps/#{app_id}/builds/#{build_id}/review"
-          req.body = build.to_json
-          req.headers['Content-Type'] = 'application/json'
-        end
-        response.body
-        require 'pry'; binding.pry
-        puts ''
-      rescue => e
-        require 'pry'; binding.pry
-        puts ''
+      response = request(:post) do |req|
+        req.url "providers/#{provider_id}/apps/#{app_id}/builds/#{build_id}/review"
+        req.body = build.to_json
+        req.headers['Content-Type'] = 'application/json'
       end
+      response.body
+      require 'pry';        puts ''
+    rescue => e
```

Comment:
Is this valid to have a `rescue` with no `begin`?  Also, we can get rid of this rescue for now, it was just there for some debugging we were doing today.